### PR TITLE
Module 1 Week 1 Portfolio Setup Skeleton

### PR DESCRIPTION
**Pull request by the-catalystmc**
For this branch:
Wrote the necessary code for index.html and style.css. 
ensured that the code did not have any errors.
Duplicated the portfolio as best I could.

### CODE REVIEW

**Comment by the-catalystmc** [comment]
I am lost as to how to place all these ellipses.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   
+    <div class="flex-container">
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo">My Logo</li>
+                
+            </ul>
+        </nav>
+    </header>
```
Comment (arslanbisharat) [inline index.html]: - [x] There should be a hamburger button on the right side which is missing in your design. 
Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <title>Portfolio</title>
+</head>
+
+<body>
+   
+    <div class="flex-container">
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo">My Logo</li>
```
Comment (arslanbisharat) [inline index.html]: - [x] I assume this a kind of name logo. I suggest using a-tag in addition to the li-tag to make it a link.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
```
Comment (arslanbisharat) [inline style.css]: -[ ] Please increase the distance between the icons using the CSS given in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <img src="images/Vectori.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorn.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorm.svg" alt="M logo" class="mlogo">
+        <img src="images/Vectorgit.svg" alt="Github logo" class="github">
+        <img src="images/Vectorpeace.svg" alt="Peace Sign" class="peace-sign">
+    </div>
+</div>
+</body>
+
+</html>
```
Comment (arslanbisharat) [inline index.html]: - [x] You were required to create the mobile website but it looks like a browser website. Please have a look at the design again and work on it. 
Capture

File: index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    </header>
+    <div class="top-images">
+        <img src="images/Ellipse3.png" alt="ellipse" class="ellipse-3">
+        <img src="images/Rectangle4.png" alt="ellipse" class="rectangle-4">
+        <img src="images/Ellipse11.png" alt="ellipse" class="ellipse-11">
+    </div>
+   
+    <section class="greet-section"> 
+ 
+        <h2>I’m Mariane Glad to see you!</h2>
```
Comment (arslanbisharat) [inline index.html]: - [x] **I’m Mariane Glad to see you!** should be enclosed in h1-tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        <p class="connect">LET’S CONNECT</p>
+    </section>
+    <div class="connect-links">
+        <img src="images/Vectortweet.svg" alt="The twitter logo" class="twitter">
+        <img src="images/Vectori.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorn.png" alt="Linked in logo" class="linkedin">
+        <img src="images/Vectorm.svg" alt="M logo" class="mlogo">
+        <img src="images/Vectorgit.svg" alt="Github logo" class="github">
+        <img src="images/Vectorpeace.svg" alt="Peace Sign" class="peace-sign">
+    </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] These should be icons, not images. You can use font awesome for the purpose. 
- [x] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

File: README.md [hunk 7]
```diff
@@ -1,2 +1,24 @@
...
+
+## About This Project
+
+This project is a portfolio website created using HTML and CSS. It is linted to meet the required standards and the unecessary files are ignored. This site is part of an activity that aids developers in creating their personal portfolio through a few sreps.
+
+## Built With
+HTML & CSS
+
+## Getting Started
+There are no installations required. Simply clone or download the zip files and open the index.html to open the webpage.
```
Comment (arslanbisharat) [inline README.md]: [ ] [Optional] You can also add how to clone this project here.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,24 @@
...
+
+## Authors
+GitHub: @the-catalyst
```
Comment (arslanbisharat) [inline README.md:15]: [ ] [Optional] You can add more links here as you added in your project.

File: README.md [hunk 9]
```diff
@@ -1,2 +1,24 @@
...
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
```
Comment (arslanbisharat) [inline README.md]: [ ] [Optional] You can try adding issue page link here.

**Review by arslanbisharat — COMMENTED** [review]
# Changes Requested, 

Hi @the-catalystmc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+        
+    </section>
+    <div class="social">
+        <ul>
+            <li><a href="#"><img src="images/Vectortweet.svg" alt="social media image"></a></li>
+            <li><a href="#"><img src="images/Linkedin.svg" alt="social media image"></a></li>
+            <li><a href="#"><img src="images/Vectorm.svg" alt="social media image"></a></li>
+            <li><a href="#"><img src="images/Vectorgit.svg" alt="social media image"></a></li>
+            <li><a href="#"><img src="images/Vectorpeace.svg" alt="social media image"></a></li>
+        </ul>>
```
Comment (Temmarie) [inline index.html]: - [x] Remove the `>` from this line.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="flex-container">
+    <section class="greet-section"> 
+        <img src="images/background.svg" alt="background image" class="banner">
+        <h1>I’m Mariane Glad to see you!</h1>
+    
+        <p class="info-section">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        <p class="connect">LET’S CONNECT</p>
```
Comment (Temmarie) [inline index.html]: - [x] As this is an invitation to connect, it's only reasonable that it is a link. It should also use a header tag. Heading tags indicate headings on a webpage by using code to tell a web browser how to display content. That's why and how they organize your content into a format that's easy to read. In addition to general structure and readability, heading tags help improve accessibility for people who can't easily read screens. 
- [x] Refer to the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to add the necessary stylings. Pay attention to the `font-styles/font-size, colours, icon-sizes`. 

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @the-catalystmc    ! :wave: :wave:

#### You have done really well in implementing the previously required changes.
- You used FlexBox ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- Consider using[ Font Awesome](https://fontawesome.com/v4.7/icons/) icons for your project instead of images.

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html (hunk 11 above)
Comment (henatan99) [reply index.html]: This issue is not addressed. Kindly make the suggested changes by the previous TSE. :pray:

File: index.html [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+        <ul>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectortweet.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Linkedin.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorm.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorgit.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorpeace.svg" alt="social media image"></a></li>
+        </ul>
+        
+    </header>
```
Comment (henatan99) [inline index.html]: - [ ] OPTIONAL: I don't think more than one header on a page is a good HTML practice. You may consider a div-tag here and better wrap it in the section element. :pray:

File: index.html [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+<body>
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo"><a href="#">My Logo</a></li>
+                <li><img src="images/Menu.svg" alt="menu image" class="menu-image"></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="flex-container">
```
Comment (henatan99) [inline index.html]: - [x] OPTIONAL: You may consider this tag more appropriately to be a main-tag. :pray:

File: index.html [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+    
+        <p class="info-section">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        
+        
+    </section>
+    <header class="social">
+        <p class="connect">LET’S CONNECT</p>
```
Comment (henatan99) [inline index.html]: - [x] You did not make the requested changes by the previous TSE for this one. You should use hx-tag and a-tag to make it a heading link. Please make sure to make this mandatory change before you submit for another review.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi the-catalystmc, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Hi @the-catalystmc, 

I think there are some mistakes about the 3 reviews. It means that you have (one review and two re-review), but since, you get confused I made this as an exceptional case and it won't happen again.
From next time if you pass 3 reviews and still some changes required, you have to repeat the week. You can check all this information [here](https://microverse.zendesk.com/hc/en-us/articles/1500004088561).

Now, let's celebrate this project completion :confetti_ball: 
### Status: Approved ✔️

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**